python/cugraph/graph/c_graph.pyx

```diff
@@ -432,27 +445,51 @@ class Graph:
             return g.adjList.offsets.size - 1
         elif g.transposedAdjList:
             return g.transposedAdjList.offsets.size - 1
-        else:
+        elif g.edgeList:
+            # This code needs to be revisited when updating gdf_graph. Users
+            # may expect numbrer_of_vertcies() as a cheap query but this
```

Comment:
Agreed, I created #241 for this. I don't expect many users to upload an edge list and directly ask for the number of vertices before actually running any analytics. So it would be nice to have but I don't see it as a top priority.